mergify_engine/github_events.py

```diff
@@ -144,6 +154,7 @@ async def filter_and_dispatch(
         repo_id = event["repository"]["id"]
         repo_name = event["repository"]["name"]
         pull_number = event["pull_request"]["number"]
+        store_active_users(redis_cache, event["repository"])
```

Comment:
You shouldn't call this function at different places. Ultimately it should be only called from only one place.